app/event/send-failure-event.js

```diff
@@ -0,0 +1,21 @@
+const config = require('../config')
+const { EventPublisher } = require('ffc-pay-event-publisher')
+const { SOURCE } = require('../constants/source')
+const { RECEIVER_CONNECTION_FAILED } = require('../constants/events')
+
+const sendFailureEvent = async (filename, error) => {
+  if (config.useEvents) {
+    const event = {
+      source: SOURCE,
+      type: RECEIVER_CONNECTION_FAILED,
+      subject: filename,
+      data: {
+        message: error.message
+      }
+    }
+    const eventPublisher = new EventPublisher(config.eventsTopic)
+    await eventPublisher.publishEvent(event)
```

Comment:
If DAX service bus is down we want to send an event with the error.  If our service bus is also down then that error currently causes the application to crash.  Worth understanding if that's just because we're not awaiting the sending in the error handleer.  Kubernetes will recover it in time, but worth understanding if there's another option.

Reply:
I've done some checks locally after adding the await in - it no longer appears to crash. Locally I receive an error each time it fails to connect (I've deleted my sandpit service bus to simulate what happened in prod). 
![image](https://github.com/DEFRA/ffc-pay-file-receiver/assets/60177449/6f247575-b0b8-4977-b1b8-670734a340d7)
Appears to try every 30 seconds ish